projects/snapshot_bot/config.h

```diff
@@ -0,0 +1,320 @@
+#pragma once
+
+// Standard C++ includes
+#include <chrono>
+#include <map>
+#include <string>
+
+// OpenCV
+#include <opencv2/opencv.hpp>
+
+// BoB robotics includes
+#include "net/connection.h"
+#include "third_party/path.h"
+#include "third_party/units.h"
+
+//------------------------------------------------------------------------
+// Config
+//------------------------------------------------------------------------
+class Config
+{
+    using Milliseconds = std::chrono::duration<double, std::milli>;
```

Comment:
I tried this and it definitely doesn't seem to work. Besides from which, I'm using these with ``std::chrono::high_resolution_clock`` et al. rather than with physical units stuff so this **seems** the more consistent choice?

Reply:
Ok, let's leave it as is then. There's probably not much point in using the units library here anyway.